kubernetes/client/ws_client.py

```diff
@@ -23,22 +25,17 @@
 
 class WSClient:
     def __init__(self, configuration, url, headers):
-        self.messages = []
-        self.errors = []
-        websocket.enableTrace(False)
-        header = None
+        enableTrace(False)
+        header = []
+        self._connected = False
+        self._stdout = ""
+        self._stderr = ""
+        self._all = ""
 
         # We just need to pass the Authorization, ignore all the other
         # http headers we get from the generated code
         if 'Authorization' in headers:
```

Comment:
Do we need to be concerned that headers is None or that there is no `Authorization` in it?